handbook/support.md

```diff
@@ -0,0 +1,13 @@
+# Requesting Support
+
+## Community Forum
+If you have any question like how to do ____ i.e. how to setup SFTP in EasyEngine, you should post in [community forum](https://community.easyengine.io/). There are helpful folks around who might be able to guide you 🕵🏻‍♂️️.
+
+## Slack
+For sharing feedback on EasyEngine and for discussion related to EasyEngine development, you can come by and say hello to us on slack :wave: [![Join EasyEngine Slack Channel](http://slack.easyengine.io/badge.svg)](http://slack.easyengine.io/)
```

Comment:
Please don't encourage slack usage at all. It's like we do have Slack. We often discuss things there. You are welcome to join. But if you expect core team to take action or reply on any feedback, its better be posted in community forum.